warehouse/admin/views/organizations.py

```diff
@@ -214,43 +209,189 @@ def organization_detail(request):
 
 
 @view_config(
-    route_name="admin.organization.approve",
+    route_name="admin.organization_application.list",
+    renderer="admin/organization_applications/list.html",
+    permission="moderator",
+    uses_session=True,
+)
+def organization_applications_list(request):
+    q = request.params.get("q", "")
```

Comment:
I double-checked this after being bit yesterday, and we allow `/admin/*` through the VCL, woohoo!